password.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+readonly class Password
+{
+    public string $password;
+    public function __construct(string $password)
+    {
+        $uppercase = str_contains('[A-Z]', $password);
+        $number = str_contains('[0-9]', $password);
+
+        if (strlen($password) < 8 || $uppercase || $number) {
+            echo 'Password should be at least 8 characters in length
```

Comment:
Remove this echo, you should not echo from within an object.

Reply:
Instead set the password to the empty string, and display the message on registration.php